### ISSUE TITLE
Nightmare crashes Node on Windows 10

### ISSUE THREAD

**Issue by bernardodesousa** [segmentio/nightmare#1476]
Since this morning, my Nightmare projects says:

> Assertion failed: frame_header.payload_length == sizeof xfer_info, file src\win\pipe.c, line 1709

Things that I tried to solve it to no avail:
1) Delete node_modules folder from the project, npm install and run again;
2) Uninstall Node, delete npm cache, reinstall node;
3) Create a brand new test Nightmare project using the first example in the readme file, the one that goes to DuckDuckGo and makes a simple search;
4) Edit the example project so it only visits the page and does nothing else.

It all ends up with the same problem. I tried on a Linux machine and it works fine. It's just here on my Windows 10 box that it happens. Also doens't happen on my other Node projects.

I'm using the latest stable versions of Node (10.6.0) and Nightmare (3.0.1).

Thanks for any help.

**Comment by ljmerza** [comment]
I think it has something to do with the latest nodejs version. I downgraded from v10.6.0 to 10.5.0 and everything works now.

**Comment by akaleeroy** [comment]
Not fixed in nodejs v10.7.0.

Error just moved 1 line
> Assertion failed: frame_header.payload_length == sizeof xfer_info, file src\win\pipe.c, line 1708

`choco install nodejs --version 10.5.0 --allow-downgrade` will not work.

**Comment by AndyDBell** [comment]
**Not fixed in Node 10.8.0**
Windows Version 10.0.17134 Build 17134
nightmare 3.0.1
Assertion failed: frame_header.payload_length == sizeof xfer_info, file src\win\pipe.c, line 1708

**Comment by aksinghnet** [comment]
I agree with @ljmerza - it has something to do with some change in Node v10.6.0.

**Temp. solution:** I uninstalled v10.7.0 and installed v10.5.0, and it just worked.

Hope it helps. Thank you.

**Comment by ljmerza** [comment]
You can also try installing the latest LTS version of node 8.x

**Comment by kmiterror** [comment]
Had the same problem with nigthmare 2.10 and node 10.6
I did some tests and it works ok with node 10.1 and the latest 10.13 so it looks like it's fixed now.
